src/encoded/tests/data/inserts/file.json

```diff
@@ -4709,5 +4709,24 @@
         "file_size": 48901763,
         "output_type": "sparse peak count matrix",
         "lab": "/labs/barbara-wold/"
+    },
+    {
+        "_test": "functional conservation scores for cross-species annotation",
+        "status": "in progress",
+        "submitted_file_name": "human_elements_with_conservation_scoring.bed.gz",
+        "uuid": "87b9a18c-2e9c-4ba2-a8b4-e5a0897a32cc",
+        "submitted_by": "facilisi.tristique@potenti.vivamus",
+        "md5sum": "3904fa99de87d37fc36bfa282c349cdb",
+        "accession": "ENCFF000CFC",
+        "file_format": "bed",
+        "file_format_type": "bed9",
+        "award": "/awards/U54HG006998/",
+        "assembly": "GRCh38",
+        "genome_annotation": "V19",
+        "dataset": "/experiments/ENCSR000CFC/",
+        "file_size": 370148294,
+        "output_type": "functional conservation scores",
```

Comment:
can we change it to "functional conservation quantifications"?

Reply:
Changed